src/Symfony/Component/Console/Formatter/OutputFormatter.php

```diff
@@ -215,6 +215,8 @@ private function createStyleFromString(string $string)
                 $style->setForeground($match[1]);
             } elseif ('bg' == $match[0]) {
                 $style->setBackground($match[1]);
+            } elseif ('hyperlink' === $match[0]) {
```

Comment:
what about using "href"?

Reply:
Dunno, `<href>` seems weird to me. I can change though if more people like this idea